docs/_getting_started/modules.md

```diff
@@ -210,20 +232,23 @@ load("rstudio")
 prepend_path("PATH","/software/free/R/R-4.2.1/bin")
 ```
 
-Linux modules only modify your current working environment which means that if you lose connection to the Yens or close your terminal window,
-you will need to reload the modules. But all of the libraries or packages that you have installed as a user will persist and 
-only need to be installed once.
+!!! Important
```

Comment:
I think we can cut this warning. We have a separate R page about R packages and libraries